### ISSUE TITLE
The type for VID & PID should not be "short"

### ISSUE THREAD

**Issue by andyhau** [LibUsbDotNet/LibUsbDotNet#91]
Instead of using "**short**", they should be "**ushort**" in order to allow all possible VIDs and PIDs. Here's one example:

![vid_pid](https://user-images.githubusercontent.com/9137067/62213039-f8f9cf00-b399-11e9-9bcf-84bb38614417.png)

 
- **MonoLibUsbApi.cs** - proposed change
```csharp
[DllImport(LIBUSB_DLL, CallingConvention = CC, SetLastError = false, EntryPoint = "libusb_open_device_with_vid_pid")]
private static extern IntPtr OpenDeviceWithVidPidInternal([In]MonoUsbSessionHandle sessionHandle, ushort vendorID, ushort productID);
```

- **MonoLibUsbApiHelpers.cs** - proposed change
```csharp
public static MonoUsbDeviceHandle OpenDeviceWithVidPid([In]MonoUsbSessionHandle sessionHandle, ushort vendorID, ushort productID)
{
    IntPtr pHandle = OpenDeviceWithVidPidInternal(sessionHandle, vendorID, productID);
    if (pHandle == IntPtr.Zero) return null;
    return new MonoUsbDeviceHandle(pHandle);
}
```
After changing it to "ushort", seems to work fine.

**Comment by bryancostanich** [comment]
I just ran into this issue as well, can confirm it should be ushort.

**Comment by mo10** [comment]
See: https://github.com/LibUsbDotNet/LibUsbDotNet/issues/10 
May be helpful.

**Comment by mcuee** [comment]
This should be fixed by #215.
